### DISCUSSION TITLE
Stuck on Sandbox Splash Screen

### DISCUSSION THREAD

**Discussion by devos91** [gucci-on-fleek/lockdown-browser#38] [Q&A]
I am able to launch Sandbox but it gets stuck on the sandbox splashscreen.
![Screenshot 2023-10-24 005202](https://github.com/gucci-on-fleek/lockdown-browser/assets/123400201/db1d314a-ea60-423c-bc0d-5b548ee61f4d)

**Comment by gucci-on-fleek** [comment]
Hmm, that's really weird. Maybe try rebooting? If that doesn't work, launch a regular version of the _Sandbox_ from the “Start Menu”. If that still doesn't work, try running
```powershell
sfc /scannow
dism /online /cleanup-image /restorehealth
sfc /scannow
```
then reboot and launch the _Sandbox_ from the “Start Menu” again. And if that still doesn't work, then I'm out of ideas.
